pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
```

Comment:
I've re-ordered the commands so that the top 3 commands are as follows:

1. Display a list of hosts belonging to a group (simplest, not requiring a network connection and has the smallest output)
2. Ping a group of hosts
3. Display facts about a group of hosts (least simple, returning lots of data back and requiring a network connection)